lib/datasource/helm/index.ts

```diff
@@ -23,12 +23,16 @@ export const defaultConfig = {
   },
 };
 
+export interface RepositoryData extends ReleaseResult {
+  name: string;
```

Comment:
For what we need this?

Reply:
The cache is for the whole helm repo, so it does a `find` based on `name`. It should be refactored from being an array of packages to being a Record

Reply:
Done